qa/rpc-tests/keypool.py

```diff
@@ -70,9 +70,11 @@ def run_test(self):
             assert(e.error['code']==-12)
 
         # refill keypool with three new addresses
-        nodes[0].walletpassphrase('test', 12000)
+        nodes[0].walletpassphrase('test', 1)
         nodes[0].keypoolrefill(3)
-        nodes[0].walletlock()
+        # test walletpassphrase timeout
+        time.sleep(1)
```

Comment:
Concept ACK, but setting a timeout of 1 and then waiting a second is too close. A slight mistiming would result in a race condition, what about waiting 2 seconds instead?